pkgs/stdenv/generic/setup.sh

```diff
@@ -1121,7 +1121,10 @@ substituteAllInPlace() {
 # the environment used for building.
 dumpVars() {
     if [ "${noDumpEnvVars:-0}" != 1 ]; then
-        install -m 0600 <(export 2>/dev/null) "$NIX_BUILD_TOP/env-vars" || true
+        # Darwin do not like install(1) being called with /dev/stdin or fd from process substitution
+        # so we first create the file and then write to it
```

Comment:
Just some small grammar changes on the comment
```suggestion
        # Darwin does not like install(1) being called with /dev/stdin or fd from process substitution
        # so first we create the file and then write to it
```